MonoGame.Framework/Graphics/Texture.cs

```diff
@@ -4,14 +4,18 @@
 
 using System;
 using System.Diagnostics;
+using System.Threading;
 
 namespace Microsoft.Xna.Framework.Graphics
 {
-	public abstract partial class Texture : GraphicsResource
+	public abstract partial class Texture : GraphicsResource, IComparable<Texture>
 	{
 		internal SurfaceFormat _format;
 		internal int _levelCount;
 
+        private readonly int _comparisonCode = Interlocked.Increment(ref _lastComparisonCode);
+        private static int _lastComparisonCode;

```

Comment:
I'm not sure if initializing it to `Int32.MinValue` would be of any real benefit... If you have created over two billion `Texture` instances, a change in sort order would be the least of your problems :)